packages/clarity-js/types/layout.d.ts

```diff
@@ -12,12 +12,17 @@ export const enum Source {
 }
 
 export const enum InteractionState {
-    Rendered = 0,
-    Visible = 10,
+    None = 16,
     Clicked = 20,
     Input = 30
 }
 
+export const enum RegionVisibility {
+    Rendered = 0,
+    Visible = 10,
+    ScrolledToEnd = 13
```

Comment:
Typo or you want an unlucky number by design? :-)

Reply:
By design - trying to have some wiggle room between interaction state and visibility state right now - since we use it for backward compatibility